### ISSUE TITLE
[upgrade] [rfe] display timeout status during output

### ISSUE THREAD

**Issue by fabbione** [ClusterLabs/anvil#382]
striker-update-cluster -vv --log-secure --clear-cache --reboot --yes --timeout 1800

[snip]

would be nice to see how much time is left before timeout triggers.

**Comment by digimer** [comment]
```
striker-update-cluster -vv --log-secure --reboot --timeout 1h -y
...
[ Note ] - [17:11:51] - It is normal for the job to show '0' progress until the database access is restored.
- Waiting for another: [58 m 55 s], will check again shortly.
```

Done, example output using '--timeout 1h'. Fixed in ed480cf

**Comment by fabbione** [comment]
Fixed in PR #376
